gateway/v8.ts

```diff
@@ -1624,3 +1625,7 @@ interface MessageReactionRemoveData {
 	guild_id?: Snowflake;
 }
 // #endregion Shared
+
+type Nullable<T> = {
```

Comment:
I'd rename this to `NullableProperties`, and maybe move it to a common place, seeing this type is duplicated in the v9 file as well.

Reply:
Can go in utils/internals.ts, and never get exported publicly

Reply:
the `Nullable` name was already used before and is consistent with the built-in `Required` and `Optional`, but I've moved it